docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -22,19 +22,27 @@ To connect to Databricks Delta Lake, you need:
 - Network access to the HMS from the coordinator and workers. Port 9083 is the
   default port for the Thrift protocol used by the HMS.
 - Data files stored in the [Parquet file format](parquet-format-configuration)
-  on a [supported file system](delta-lake-file-system-configuration).
+  on a [supported file system](delta-lake-file-system-configuration). You must
+  select and configure the desired file system in your catalog `.properties`
+  file.
 
 ## General configuration
 
 To configure the Delta Lake connector, create a catalog properties file
-`etc/catalog/example.properties` that references the `delta_lake`
-connector and defines a metastore. You must configure a metastore for table
-metadata.  If you are using a {ref}`Hive metastore <hive-thrift-metastore>`,
-`hive.metastore.uri` must be configured:
+`etc/catalog/example.properties` that references the `delta_lake` connector and
+defines a metastore. You must configure a metastore for table metadata. If you
+are using a {ref}`Hive metastore <hive-thrift-metastore>`, `hive.metastore.uri`
+must be configured.
+
+By default, no file system is selected for your object storage catalog. You must
+select and configure from one of the [supported file
+systems](file-system-configuration). Replace the `x` in the `fs.x.enabled`
+configuration property with your desired file system.
```

Comment:
Rather than include this here, I would make this a code comment like

```
...
//replace the following property with the desired file system implementation
fs.x.enabled=true
```